src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -70,6 +70,13 @@ def get(self, request: Request, organization: Organization) -> Response:
         query = request.GET.get("query")
         span_ops = request.GET.getlist("spanOp")
 
+        try:
+            per_suspect = int(request.GET.get("perSuspect", 4))
```

Comment:
If the default is 4, can we bump max to .. maybe 10? for testing? Or were you adding this to allow asking for just 1?

Reply:
bumped the max to 10 👍